pages/common/createdb.md

```diff
@@ -0,0 +1,16 @@
+# createdb
+
+> Create a PostgreSQL database.
+> More information: <https://www.postgresql.org/docs/current/app-createdb.html>.
+
+- Create a database owned by the current user:
+
+`createdb {{database_name}}`
+
+- Create a database owned by a specific user, with an optional description:
```

Comment:
```suggestion
- Create a database owned by a specific user with a description:
```

The command is based around having a description. Plus after removing the word optional, it reads better without the comma.